port_ocean/core/ocean_types.py

```diff
@@ -30,6 +38,7 @@ class EntitySelectorDiff(NamedTuple):
 class CalculationResult(NamedTuple):
     entity_selector_diff: EntitySelectorDiff
     errors: list[Exception]
+    misconfigured: dict[str, str] = field(default_factory=dict)
```

Comment:
I wonder if `misconfigured` is specific enough? as I am not sure I totally understand what it relates to as misconfigured

maybe `misonfigured_entity_keys` ?